app/controllers/map_controller.rb

```diff
@@ -16,7 +16,11 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    finder = "#{params[:name]} #{params[:date]}".parameterize
```

Comment:
Nice, this makes sense. Is there a way to create a new `.find` method such that the parameters may be passed as comma-delimited arguments, like `.find(params[:name], params[:date])`? This would be simpler and we could later generate the string in the new `.find` method and pass it to the old `.find` method (the `friendly_id` one which accepts a string). If that's too complicated, at least we could make a `.find_map` method or something which formats the string and submits a `.find` call. 

I ask because i see the string construction at multiple points here.